### ISSUE TITLE
9.1 errors

### ISSUE THREAD

**Issue by Cluuey** [kemayo/wow-handynotes-shadowlandstreasures#4]
I thought I'd have a look on the PTR tonight, probably should have looked earlier!
Instead of opening a ticket for each bug I'll post them all in here, there are six, I don't think there's a way of splitting them into their own little code blocks.

If I comment out the `ns.currencies` section of `handler.lua` the first error stops but I don't know what the consequences of that are, I only did it to see what would happen with the other errors.

Bastion, Ardenweald, Maldraxxus and Revendreth errors look like they're pointing to quests which have been removed, at least that's what my investigation led me to believe. I just changed the Wowhead URL to have the quests specified and they don't exist, it's quite possible I'm missing something here.

The Maw quests still exist as does the achievement, _I'm way out of my depth here so I'll stop trying to help._

```1x ...eHandyNotes_ShadowlandsTreasures\handler.lua:18: attempt to concatenate global 'ANIMA' (a nil value)
[string "@HandyNotes_ShadowlandsTreasures\handler.lua"]:18: in main chunk

Locals:
myname = "HandyNotes_ShadowlandsTreasures"
ns = <table> {
 map_spellids = <table> {
 }
 HL = <table> {
 }
}
HandyNotes = <table> {
 SetDefaultModuleLibraries = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:367
 GetContinentZoneList = <function> defined @HandyNotes\HandyNotes.lua:229
 GetMapFiletoMapID = <function> defined @HandyNotes\HandyNotes.lua:261
 Enable = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:294
 RegisterChatCommand = <function> defined @HandyNotes\Libs\AceConsole-3.0\AceConsole-3.0.lua:85
 EnableModule = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:332
 modules = <table> {
 }
 GetModule = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:210
 IterateEmbeds = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:442
 plugins = <table> {
 }
 defaultModuleLibraries = <table> {
 }
 UnregisterChatCommand = <function> defined @HandyNotes\Libs\AceConsole-3.0\AceConsole-3.0.lua:111
 Printf = <function> defined @HandyNotes\Libs\AceConsole-3.0\AceConsole-3.0.lua:69
 OnProfileChanged = <function> defined @HandyNotes\HandyNotes.lua:651
 SetDefaultModulePrototype = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:409
 UpdateMinimapPlugin = <function> defined @HandyNotes\HandyNotes.lua:421
 name = "HandyNotes"
 IsEnabled = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:451
 orderedModules = <table> {
 }
 DisableModule = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:350
 UpdateWorldMapPlugin = <function> defined @HandyNotes\HandyNotes.lua:406
 baseName = "HandyNotes"
 OnDisable = <function> defined @HandyNotes_Shadowlands\core\map.lua:381
 OnEnable = <function> defined @HandyNotes_Shadowlands\core\map.lua:375
 RegisterMessage = <function> defined @BugSack\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90
 GetArgs = <function> defined @HandyNotes\Libs\AceConsole-3.0\AceConsole-3.0.lua:144
 Print = <function> defined @HandyNotes\Libs\AceConsole-3.0\AceConsole-3.0.lua:54
 OnInitialize = <function> defined @HandyNotes\HandyNotes.lua:608
 UpdatePluginMap = <function> defined @HandyNotes\HandyNotes.lua:492
 UpdateMinimap = <function> defined @HandyNotes\HandyNotes.lua:482
 NewModule = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:235
 IsModule = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:468
 db = <table> {
 }
 SetDefaultModuleState = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:387
 WorldMapDataProvider = <table> {
 }
 GetMapIDtoMapFile = <function> defined @HandyNotes\HandyNotes.lua:267
 SetEnabledState = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:424
 getXY = <function> defined @HandyNotes\HandyNotes.lua:255
 enabledState = true
 UpdateWorldMap = <function> defined @HandyNotes\HandyNotes.lua:412
 RegisterEvent = <function> defined @BugSack\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90
 UnregisterAllMessages = <function> defined @BugSack\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:174
 RegisterPluginDB = <function> defined @HandyNotes\HandyNotes.lua:152
 UnregisterAllEvents = <function> defined @BugSack\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:174
 SendMessage = <function> defined @BugSack\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:59
 IterateModules = <function> defined @HandyNotes\Libs\AceAddon-3.0\AceAddon-3.0.lua:437
 UnregisterMessage = <function```

1x ...ns\HandyNotes_ShadowlandsTreasures\zones\Bastion.lua:48: attempt to call local 'path' (a nil value)
[string "@HandyNotes_ShadowlandsTreasures\zones\Bastion.lua"]:48: in main chunk

Locals:
myname = "HandyNotes_ShadowlandsTreasures"
ns = <table> {
 defaults = <table> {
 }
 options = <table> {
 }
 should_show_point = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:503
 HL = <table> {
 }
 allQuestsComplete = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:283
 itemIsKnown = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:400
 itemRestricted = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:384
 map_spellids = <table> {
 }
 SetupMapOverlay = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:617
 RouteWorldMapDataProvider = <table> {
 }
 itemIsKnowable = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:394
}
path = nil
(*temporary) = nil
(*temporary) = 1533
(*temporary) = <table> {
 53508030 = <table> {
 }
 59306080 = <table> {
 }
 58204000 = <table> {
 }
 58507150 = <table> {
 }
 52008600 = <table> {
 }
}
(*temporary) = nil
(*temporary) = <table> {
 quest = 61049
}
(*temporary) = "attempt to call local 'path' (a nil value)"


1x ...HandyNotes_ShadowlandsTreasures\zones\Ardenweald.lua:38: attempt to call field 'path' (a nil value)
[string "@HandyNotes_ShadowlandsTreasures\zones\Ardenweald.lua"]:38: in main chunk

Locals:
myname = "HandyNotes_ShadowlandsTreasures"
ns = <table> {
 defaults = <table> {
 }
 options = <table> {
 }
 should_show_point = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:503
 HL = <table> {
 }
 allQuestsComplete = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:283
 itemIsKnown = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:400
 itemRestricted = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:384
 map_spellids = <table> {
 }
 SetupMapOverlay = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:617
 RouteWorldMapDataProvider = <table> {
 }
 itemIsKnowable = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:394
}
path = nil
(*temporary) = nil
(*temporary) = 1565
(*temporary) = <table> {
 49705590 = <table> {
 }
 55902100 = <table> {
 }
 52903730 = <table> {
 }
 48203920 = <table> {
 }
}
(*temporary) = nil
(*temporary) = <table> {
 achievement = 14313
 note = "Slow-fall feather to reach the treasure"
 criteria = 50037
 quest = 61067
 atlas = "poi-door-arrow-up"
}
(*temporary) = "attempt to call field 'path' (a nil value)"


1x ...HandyNotes_ShadowlandsTreasures\zones\Maldraxxus.lua:114: attempt to call field 'path' (a nil value)
[string "@HandyNotes_ShadowlandsTreasures\zones\Maldraxxus.lua"]:114: in main chunk

Locals:
myname = "HandyNotes_ShadowlandsTreasures"
ns = <table> {
 defaults = <table> {
 }
 options = <table> {
 }
 should_show_point = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:503
 HL = <table> {
 }
 allQuestsComplete = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:283
 itemIsKnown = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:400
 itemRestricted = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:384
 map_spellids = <table> {
 }
 SetupMapOverlay = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:617
 RouteWorldMapDataProvider = <table> {
 }
 itemIsKnowable = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:394
}
path = nil
(*temporary) = nil
(*temporary) = 1536
(*temporary) = <table> {
 32702120 = <table> {
 }
 47206210 = <table> {
 }
 57607580 = <table> {
 }
 59807900 = <table> {
 }
 62405990 = <table> {
 }
 37907000 = <table> {
 }
 31707000 = <table> {
 }
 72805360 = <table> {
 }
 30702870 = <table> {
 }
 49401510 = <table> {
 }
 22503050 = <table> {
 }
 40603300 = <table> {
 }
 42302430 = <table> {
 }
}
(*temporary) = nil
(*temporary) = <table> {
 achievement = 14312
 note = "Under the platform; stand in the goo to get 10 stacks of {spell:330069} and become become {spell:330092:Plaguefallen}, then go into the cave and you'll be able to get through the pipe"
 criteria = 50074
 label = "Tunnel entrance"
 level = 60
 quest = 61474
}
(*temporary) = "attempt to call field 'path' (a nil value)"


1x ...HandyNotes_ShadowlandsTreasures\zones\Revendreth.lua:5: attempt to call field 'RegisterPoints' (a nil value)
[string "@HandyNotes_ShadowlandsTreasures\zones\Revendreth.lua"]:5: in main chunk

Locals:
myname = "HandyNotes_ShadowlandsTreasures"
ns = <table> {
 defaults = <table> {
 }
 options = <table> {
 }
 should_show_point = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:503
 HL = <table> {
 }
 allQuestsComplete = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:283
 itemIsKnown = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:400
 itemRestricted = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:384
 map_spellids = <table> {
 }
 SetupMapOverlay = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:617
 RouteWorldMapDataProvider = <table> {
 }
 itemIsKnowable = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:394
}
path = nil
(*temporary) = nil
(*temporary) = 1525
(*temporary) = <table> {
 51815963 = <table> {
 }
 31055510 = <table> {
 }
 37706920 = <table> {
 }
 79903700 = <table> {
 }
 70106000 = <table> {
 }
 74975698 = <table> {
 }
 47505530 = <table> {
 }
 38404430 = <table> {
 }
 46395817 = <table> {
 }
 68456446 = <table> {
 }
 64197265 = <table> {
 }
 29703720 = <table> {
 }
 61565865 = <table> {
 }
 63007210 = <table> {
 }
 69357799 = <table> {
 }
 57304330 = <table> {
 }
}
(*temporary) = "attempt to call field 'RegisterPoints' (a nil value)"


1x HandyNotes_ShadowlandsTreasures\zones\Maw.lua:212: attempt to call local 'path' (a nil value)
[string "@HandyNotes_ShadowlandsTreasures\zones\Maw.lua"]:212: in main chunk

Locals:
myname = "HandyNotes_ShadowlandsTreasures"
ns = <table> {
 defaults = <table> {
 }
 options = <table> {
 }
 should_show_point = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:503
 HL = <table> {
 }
 allQuestsComplete = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:283
 itemIsKnown = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:400
 itemRestricted = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:384
 map_spellids = <table> {
 }
 SetupMapOverlay = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:617
 RouteWorldMapDataProvider = <table> {
 }
 itemIsKnowable = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:394
}
path = nil
CAVE = "Cave entrance"
(*temporary) = nil
(*temporary) = 1543
(*temporary) = <table> {
 28701205 = <table> {
 }
 19204610 = <table> {
 }
 23755340 = <table> {
 }
 42352110 = <table> {
 }
 35954155 = <table> {
 }
 28106060 = <table> {
 }
 23702140 = <table> {
 }
 45507375 = <table> {
 }
 48801830 = <table> {
 }
 49128175 = <table> {
 }
 26752930 = <table> {
 }
 20606935 = <table> {
 }
 60954805 = <table> {
 }
 25903115 = <table> {
 }
 27751305 = <table> {
 }
 32956645 = <table> {
 }
 53507950 = <table> {
 }
 19304170 = <table> {
 }
 16955100 = <table> {
 }
 37656590 = <table> {
 }
 27407150 = <table> {
 }
 30775000 = <table> {
 }
 39004120 = <table> {
 }
}
(*temporary) = nil
(*temporary) = <table> {
 achievement = 14660
 criteria = <table> {
 }
 label = "Cave entrance"
 level = 60
 quest = <table> {
 }
}
(*temporary) = "attempt to call local 'path' (a nil value)"


1x ...Ons\HandyNotes_ShadowlandsTreasures\zones\Oribos.lua:3: attempt to call field 'RegisterPoints' (a nil value)
[string "@HandyNotes_ShadowlandsTreasures\zones\Oribos.lua"]:3: in main chunk

Locals:
myname = "HandyNotes_ShadowlandsTreasures"
ns = <table> {
 defaults = <table> {
 }
 options = <table> {
 }
 should_show_point = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:503
 HL = <table> {
 }
 allQuestsComplete = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:283
 itemIsKnown = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:400
 itemRestricted = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:384
 map_spellids = <table> {
 }
 SetupMapOverlay = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:617
 RouteWorldMapDataProvider = <table> {
 }
 itemIsKnowable = <function> defined @HandyNotes_ShadowlandsTreasures\config.lua:394
}
(*temporary) = nil
(*temporary) = 1550
(*temporary) = <table> {
}
(*temporary) = "attempt to call field 'RegisterPoints' (a nil value)"

**Comment by kemayo** [comment]
Almost all of that is fallout from an error in handler.lua -- the various `attempt to call local` errors just mean that a thing they expected from that file doesn't exist.

**Comment by kemayo** [comment]
It'll be fixed as of aa636b9 - I'm not going to do a release right now, as I hope to finish the 9.1 content notes before the servers are back up. :D
